test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+PGBACKREST_BIN=/usr/local/bin
```

Comment:
The PGBACKREST_BIN variable is used as a path to the pgbackrest binary only. I suggest adding /usr/local/bin to PATH if this directory is not there. So we will be able to write `pgbackrest` instead of `$PGBACKREST_BIN/pgbackrest`